src/components/Popover/PopoverHeader.tsx

```diff
@@ -1,3 +1,10 @@
-import { PopoverHeader } from 'reactstrap';
+import type { FC } from 'react';
+import React from 'react';
+import type { PopoverHeaderProps } from 'reactstrap';
+import { PopoverHeader as InternalPopoverHeader } from 'reactstrap';
```

Comment:
[TypeScript 4.5 type modifiers on import names.](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-5.html#type-modifiers-on-import-names)
```suggestion
import { PopoverHeader as InternalPopoverHeader, type PopoverHeaderProps } from 'reactstrap';
```

Reply:
Boom!
I'll need to fix the `@typescript-eslint/consistent-type-imports` rule to see if this syntax is supported.

Reply:
Actually, this is a great suggestion, but I'm going to need to upgrade to TS 4.5 and then alter this syntax everywhere.
Saving all that for a follow-up PR.